.vscode/extensions.json

```diff
@@ -1,3 +1,18 @@
 {
-  "recommendations": ["esbenp.prettier-vscode", "ms-python.python"]
+  // Please keep this file in sync with settings in home-assistant/.devcontainer/devcontainer.json
+  "recommendations": [
+    // common extensions
+    "visualstudioexptteam.vscodeintellicode",
+    "redhat.vscode-yaml",
+    "esbenp.prettier-vscode",
+    "GitHub.vscode-pull-request-github",
+    "eamodio.gitlens",
+    "bungcip.better-toml",
+    "streetsidesoftware.code-spell-checker",
+    // python extensions
+    "ms-python.python",
+    "ms-python.vscode-pylance",
+    "ms-python.pylint",
+    "ms-python.isort"
```

Comment:
These are already a dependency of `ms-python.python` and thus not needed.

Reply:
Microsoft is breaking out all of the Python linters into their own extension. So, the preferred way is to _not_ use the one that comes with the Python extension anymore. The standalone one is better.

https://marketplace.visualstudio.com/items?itemName=ms-python.isort

Reply:
That is currently in preview and this should not be used.